server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,8 @@
+export const cleanUserInput = (formInput) => {
```

Comment:
consider the use of a `reduce` function here to delete blanks. I might also consider calling this function something more intentional, like `deleteBlanks`